gaps/gap-3_mid_agreement_payments/gap-3_mid_agreement_payments.md

```diff
@@ -24,13 +24,15 @@ after it was accepted. This specification describes, how Requestor and Provider
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| `golem.com.scheme.payu.debit-note-interval-sec?` | Number (uint32) [Negotiable] | Minimum time (in seconds) between subsequent Debit Notes. The Provider must not issue the next Debit Note before the interval period. The Requestor ignores Debit Notes issued before interval elapses. |
+| `golem.com.scheme.payu.debit-note.interval-sec?` | Number (uint32) [Negotiable] | Minimum time (in seconds) between subsequent Debit Notes related to a single Activity. The Provider must not issue the next Debit Note before the interval period. The Requestor ignores Debit Notes issued before interval elapses. |
 | `golem.com.scheme.payu.payment-timeout-sec?` | Number (uint32) [Negotiable] | Maximum time to receive payment for any Debit Note. This time is counted from Debit Note’s issue date. |
 
 - Requestor can ignore Debit Note, if it came before expected interval
   - If Requestor is spammed with Debit Notes by Provider, he is allowed to break Agreement stating (in TerminateAgreement) that the **Reason** for termination is Provider's perceived misconduct. He should set `golem.requestor.code` field in **Reason** structure to `TooManyDebitNotes`.
 - In the mechanism suggested above a Debit Note shall have PaymentDueDate set, and the values must be set so that they correspond with the agreed `payment-interval-sec` value.
```

Comment:
```suggestion
- In the mechanism suggested above a Debit Note shall have PaymentDueDate set, and the values must be set so that they correspond with the agreed `golem.com.scheme.payu.payment-timeout-sec? ` value.
```